test/Dialect/FIRRTL/parse-basic.fir

```diff
@@ -1689,12 +1689,42 @@ circuit BasicProps :
 
   ; CHECK-LABEL: module private @Path
   module Path :
-    ; CHECK-SAME: in %a: !firrtl.path
-    ; CHECK-SAME: out %b: !firrtl.path
-    input a : Path
-    output b : Path
-    ; CHECK: firrtl.propassign %b, %a : !firrtl.path
-    propassign b, a
+    output psimple : Path
+    output pbundle : Path
+    output pvector : Path
+    output pinst : Path
+    output pinstport : Path
+    output pport : Path
+    ; CHECK: in %port: !firrtl.uint<8> sym @sym_3
```

Comment:
capture expressions for symbols?

Reply:
switched to using captures for everything